app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -252,7 +252,6 @@ class ExplorationActivityTest {
   }
 
   @Test
-  @DisableAccessibilityChecks // TODO(#3251): Enable AccessibilityChecks
```

Comment:
All these tests are passing now but `3 tests` are still failing for which I have kept the annotation `DisableAccessibilityChecks `.

**When all `DisableAccessibilityChecks ` are removed and not fixed.**
<img width="1528" alt="Screenshot 2021-06-24 at 1 29 59 AM" src="https://user-images.githubusercontent.com/9396084/123170619-5a4c4f00-d498-11eb-97c2-1f87ce9c7ad1.png">


**When all `DisableAccessibilityChecks` are fixed except 3.**
<img width="1560" alt="Screenshot 2021-06-24 at 2 12 23 AM" src="https://user-images.githubusercontent.com/9396084/123170642-620bf380-d498-11eb-8edb-d9529e31c373.png">